.gitignore

```diff
@@ -182,6 +182,6 @@ override.tf.json
 .terraformrc
 terraform.rc
 
-#ignore local env
-.env.local
-.env.my_hosted_flex
+# Ignore local env(s), allowing only the example env
+.env.*
+!.env.example
```

Comment:
This will prevent users from committing overrides to the autogenerated environments for hosted flex deployments though

Reply:
Just updated the docs so that users may do this if needed. Per our Slack discussion, hopefully we can work towards eliminating the need altogether :)